src/pages/index.js

```diff
@@ -210,17 +435,60 @@ function handleProfileFormSubmit(inputValues) {
   profileCardPopup.closeModal();
 }
 
-function handleCardFormSubmit(cardValues) {
-  const name = cardValues.title;
-  const link = cardValues.url;
-  renderCard({ name, link }, cardListEl);
-  // contentFormElement.reset();
-  // const submitButton = document.querySelector("#modalContentSubmit");
-  // submitButton.classList.toggle("modal__button_disabled");
-  // submitButton.setAttribute("disabled", "true");
-  newCardPopup.closeModal();
+// function handleCardFormSubmit(cardValues) {
+//   const name = cardValues.title;
+//   const link = cardValues.url;
+//   const submitButton = document.getElementById("modalContentSubmit");
+//   const originalButtonText = submitButton.textContent;
+//   // Change button text to "Saving..."
+//   submitButton.textContent = "Saving...";
+//   api
+//     .addNewCard({
+//       name,
+//       link,
+//     })
+//     .then(() => addCardSection.addItem(name, link))
+//     .then(() => (submitButton.textContent = originalButtonText))
+//     .catch((err) => {
+//       console.log(err);
+//       submitButton.textContent = originalButtonText;
+//     });
+//   // contentFormElement.reset();
+
+//   // submitButton.classList.toggle("modal__button_disabled");
+//   // submitButton.setAttribute("disabled", "true");
+//   newCardPopup.closeModal();
+// }
+
+function handleCardLike(card) {
+  console.log(card.getId());
+  api
+    .addCardLike(card.getId())
+    .then(() => card.handleLikeIcon())
+    .catch((err) => {
+      console.log(err);
+    });
 }
 
+function handleCardLikeRemove(card) {
+  console.log(card.getId());
+  api
+    .removeCardLike(card.getId())
+    .then(() => card.handleLikeIcon())
+    .catch((err) => {
+      console.log(err);
+    });
+}
+
+//test
+// api
+//   .removeCardLike("65c971d1879b84001ae42ee0")
+//   .then((card) => console.log(card))
+//   .catch((err) => {
+//     console.log(err);
+//   });
+//test
+
 const config = {
```

Comment:
From the 8th sprint all constants (`initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.